cves/CVE-2018-1305.yml

```diff
@@ -109,9 +114,14 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: |
+    The code is automatically unit tested before being deployed. This
+    vulnerability was not tested though. They did not change their unit tests during
```

Comment:
Do you think the fact that they didn't test this particular part of the code before was one of the reasons why this wasn't found before release?

Reply:
I think without testing you can't know it is broken in the first place but they also probably just didn't think that load order was going to cause a security issue in the first place.